mirrord/agent/iptables/src/lib.rs

```diff
@@ -219,8 +219,7 @@ where
 
     /// List rules from other/ previous mirrord agents that exist on the IP table
     #[tracing::instrument(level = Level::TRACE, skip(ipt) ret, err)]
-    pub async fn list_mirrord_rules(ipt: IPT) -> IPTablesResult<Vec<String>> {
-        let ipt = Arc::new(ipt);
```

Comment:
Why did we have this?

Reply:
Not sure, maybe some cleanup that's overlooked.